checks/cloud/azure/network/disable_rdp_from_internet.rego

```diff
@@ -49,4 +49,7 @@ deny contains res if {
 	)
 }
 
-port_range_includes(from, to, port) if from <= port <= to
+port_range_includes(from, to, port) if {
+	from.value <= port
+	port <= to.value
+}
```

Comment:
I'm curious is this valid:

```if from.value <= port <= to.value```?

Reply:
This kind of expression doesn't work in Rego: https://play.openpolicyagent.org/p/2RS5HYLT5d